locales/en-US/app.ftl

```diff
@@ -702,6 +702,12 @@ TransformNavigator--merge-call-node = Merge Node: { $item }
 #   $item (String) - Name of the function that transform applied to.
 TransformNavigator--merge-function = Merge: { $item }
 
+# "Merge function set" transform.
+# See: https://profiler.firefox.com/docs/#/./guide-filtering-call-trees?id=merge
+# Variables:
+#   $item (String) - The count of functions merged.
```

Comment:
Why `item` and not, say, `count`?

Reply:
This is going through some shared types for the component, so it would be a more complicated implementation to change the name.

```
export type TransformLabeL10nIds = {|
  +l10nId: string,
  +item: string,
|};
```